node/bft/src/sync/mod.rs

```diff
@@ -489,11 +489,12 @@ impl<N: Network> Sync<N> {
             let commit_round = leader_certificate.round();
             let certificate_round = commit_round.saturating_add(1);
 
-            // Get the committee lookback for the commit round.
-            let committee_lookback = self.ledger.get_committee_lookback_for_round(commit_round)?;
-            // Retrieve all of the certificates for the **certificate** round.
+            // Get the committee lookback for the round just after the leader.
+            let committee_lookback = self.ledger.get_committee_lookback_for_round(certificate_round)?;
```

Comment:
Should we also name this `certificate_committee_lookback`?

Reply:
Sure. Pushed.